website/blog/2022-01-21-release.md

```diff
@@ -0,0 +1,184 @@
+
+```shell
+(venv) $ wave run
+```
+
+Note that this simplification is aimed for development only. For production deployments, you might want to pass additional configuration options to waved (not possible with `wave run`) or even switch your python ASGI server (see [deployment docs](/docs/deployment#deploying-wave-apps)).
+
+If you for, some reason, do not want to start `waved` during `wave run`, you can turn it off via `H2O_WAVE_NO_AUTOSTART` env variable.
+
+All the examples and Tour app were previously included in the release. This is not the case with `pip install` though. If you would still like to have all the examples accessible locally, just run `wave fetch` to download examples, demos, and the interactive tour.
+
+```shell
+(venv) $ wave fetch
+```
+
+You should now have access to 200+ examples locally.
+
+```
+Fetching examples and related files. Please wait...
+Downloading https://github.com/h2oai/wave/releases/download/v0.19.0/wave-0.19.0-linux-amd64.tar.gz
+Extracting...
+
+All additional files downloaded and extracted successfully!
+Examples and tour............. /home/elp/git/wave/py/wave/examples
+Demos and layout samples...... /home/elp/git/wave/py/wave/demo
+Automated test harness........ /home/elp/git/wave/py/wave/test
+Wave daemon for deployments... /home/elp/git/wave/py/wave
+```
+
+## Base URL support
+
+By default, Wave apps are all registered on `/` root path. The location your app listens on depends on the route you specify in `@app` or `site['']`. However, if you want to namespace apps or just facing complex deployment, you might want to configure it from a single place rather than on per-app basis.
+
+Launch `waved` with `-base-url` or launch `waved` with `$H2O_WAVE_BASE_URL` to set a base URL. Setting base URL to `/foo/` will allow serving an app with route `/` at `/foo/`, or an app with route `/bar` at `/foo/bar/`. Accessing `/foo/`'s parent(s) from browser/API will return 404. If a base URL is set, an app must be launched with `$H2O_WAVE_BASE_URL` set to the same base URL value.
+
+## Improved docs
+
+For the sake of time, our documentation was mostly made up of plain code examples without any extra explanations/descriptions. The [newest version](/docs/components/overview), although not yet fully complete, tries to fix that by organizing the available components according to their usecase and providing all the possible component variations with guidance on when and how they should be used. Additionally, all the documented components include accompanying screenshots so that developers know exactly how is the particular component going to render.
+
+## Component additions/improvements
+
+This release contains a lot of bugfixes and small features that are not worth mentioning from a general overview point of view. Instead, here is a list of the most significant component changes and additions.
+
+### Header
```

Comment:
- Is this list complete?
- Would be good to include links to the /widgets/xyz page that describes the widget.

Reply:
> Is this list complete?

Will check again, but these should be all the new cards/components that were added in 0.20.

> Would be good to include links to the /widgets/xyz page that describes the widget.

Agree, will do.